docfx/doxygen2yaml.cc

```diff
@@ -71,11 +71,16 @@ std::string Summary(pugi::xml_node const& node) {
   MarkdownContext ctx;
   ctx.paragraph_start = "";
   auto brief = node.child("briefdescription");
-  auto detailed = node.child("detaileddescription");
   if (!brief.empty()) {
     AppendIfBriefDescription(os, ctx, brief);
     ctx = MarkdownContext{};
   }
+  auto description = node.child("description");
+  if (!description.empty()) {
+    AppendDescriptionType(os, ctx, brief);
```

Comment:
s/brief/description/

unit test?

Reply:
Yikes, good thing you asked for a test.  That is what happens when I try to YOLO things.